faker/providers/job/el_GR/__init__.py

```diff
@@ -0,0 +1,453 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as BaseProvider
+
+
+class Provider(BaseProvider):
+    jobs = [
+        'Αγγειοπλάστης',
```

Comment:
do you have a source for this list? If so, could you include it as comment?

Reply:
there is no specific source, I was browsing the web gathering sample data to populate the job descriptions list